test/e2e/tests/ppom-blockaid-alert-erc20-approval.spec.js

```diff
@@ -45,7 +45,6 @@ async function mockInfura(mockServer) {
         methodResultVariant: 'balance',
         params: [
           {
-            accessList: [],
```

Comment:
removing this, makes the mock be triggered for the same request without `accessList`. This tweak fixed the "not displaying the ERC20 Approve screen`